main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    data = [("genre", "Western"),
+            ("genre", "Action"),
+            ("genre", "Dramma"),
+            ("actor", "George", "Klooney"),
+            ("actor", "Kianu", "Reaves"),
+            ("actress", "Scarlett", "Keegan"),
+            ("actor", "Will", "Smith"),
+            ("actor", "Jaden", "Smith"),
+            ("actress", "Scarlett", "Johansson")]
+    for item in data:
+        if item[0] == "genre":
+            Genre.objects.create(name=item[1])
+        if item[0] == "actor" or item[0] == "actress":
+            Actor.objects.create(first_name=item[1],
+                                 last_name=item[2])
```

Comment:
why dont u separate the data e.g. genres/actors and make a 2 separate loop for creating for better readability ?

Reply:
previously, I was told that it's preferable not to create multiple loops if everything can be done in one

Reply:
here is ok to have two loops since these are different entities and combining them into one loop will not affect performance. here you have 3 iterations for genres and 6 iterations for actors. Wether you put them in one loop or separate it will still be 9 iterations total